docs/CODE_STYLE.md

```
@@ -1,60 +1,85 @@
+A solid GameMaker-specific formatter, that formats most of the elements that can get untidy, is [Gobo](https://github.com/Pizzaandy/Gobo/).
+- Disable tab indent and set it to 4 spaces.
+- Set max line width to something like 9999, to prevent condition checks from breaking into unreadable multiple lines.
+- The above step will format structs into a single line, so you'll have to use a JS formatter afterwards to fix this.
 
-- You can use [this](https://beautifier.io/) JavaScript formatter with:
-  - Indent with 4 spaces.
-  - Allow unlimited newlines between tokens.
-  - Don't wrap lines.
-  - Spaces before conditional.
+You can use [this](https://beautifier.io/) JavaScript formatter to fix broken structs.
+- Indent with 4 spaces.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)